velox/exec/WindowPartition.cpp

```diff
@@ -378,7 +381,9 @@ void WindowPartition::updateKRangeFrameBounds(
         RowContainer::isNullAt(
             partitionRow,
             orderByRowColumn.nullByte(),
-            orderByRowColumn.nullMask()));
+            orderByRowColumn.nullMask()),
+        "Frame column should be NULL iff ORDER BY value is NULL for row: {}",
+        partitionRow);
```

Comment:
partitionRow pointer is not really meaningful to the user. Its likely it helped you for debugging though.  If we can print the contents of the entire row here, that would be more helpful.

But in general, this is a detail that comes from the implementation of range column search and not a user issue. So not having an error message is fine as well imo.